Uplift/Controllers/OnboardingViewController.swift

```diff
@@ -94,6 +94,7 @@ class OnboardingViewController: PresentationController {
 
         // Set Reconnect Action if couldn't fetch gyms/classes
         let retryNetworkRequest: (() -> Void) = {
+            /*
```

Comment:
Same question as earlier, following the documentation [here](https://google.github.io/swift/#file-comments), such that "Non-documentation comments always use the double-slash format (//), never the C-style block format (/* ... */)."